docs/JestObjectAPI.md

```diff
@@ -910,8 +999,24 @@ This function is not available when using legacy fake timers implementation.
 
 ### `jest.setTimeout(timeout)`
 
+
+Set the default timeout interval (in milliseconds) for all tests and before/after hooks in the test file. This only affects the test file from which this function is called.
```

Comment:
Why are you adding this text?

Reply:
I'm not sure why it is showing that I added it but I double checked in the main app and this is preexisting text.